addons/SmoothScroll/SmoothScrollContainer.gd

```diff
@@ -630,6 +630,21 @@ func scroll_to_left(duration:float=0.5) -> void:
 func scroll_to_right(duration:float=0.5) -> void:
 	scroll_x_to(self.size.x - content_node.size.x, duration)
 
+func scroll_to_fauna(duration:float=0.5) -> void:
+	if(!PlayerManager.is_player_initial_position_set):
+		return
+	
+	var room_distance: float = 66
```

Comment:
is this the distance in pixel / unit between floors ?
if yes, you can get it without hard-coding in two different ways:
- either use `_get_combined_room_height` (which will need to be renamed to `get_combined_room_height` to show it's not private anymore) and divide by the number of floors
- replicate the logic of `_get_combined_room_height` to calculate the height of a room at a floor index y

It's better to not have hard-coded values

Reply:
Ye, I'll fix that I didn't know about these variables

Reply:
I think it would be better to replicate the logic of `_get_combined_room_height` to just get the current height
because the function is already doing texture size * total height, but we can just do texture size * index (+ some offset probably)